packages/react/src/lib/accessibility/Behaviors/Dialog/dialogBehavior.ts

```diff
@@ -1,5 +1,6 @@
 import { Accessibility } from '../../types'
 import popupFocusTrapBehavior from '../Popup/popupFocusTrapBehavior'
+import * as _ from 'lodash'
 
 /**
  * @description
```

Comment:
specification with unit tests would be preferable, but if not, at least add the description

Reply:
I tried to add specs and another definition. Please take a look.